iOS/MSQASignInSwift/MSQASignInButton.swift

```diff
@@ -0,0 +1,157 @@
+//------------------------------------------------------------------------------
+//
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+//
+// This code is licensed under the MIT License.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//-----------------------------------------------------------------------------
+
+import Foundation
+import SwiftUI
+
+/// Microsoft Quick Auth sign in button.
+@available(iOS 13.0, *)
+public struct MSQASignInButton: View {
+  /// An object containing the customization to the button.
+  @ObservedObject public var viewModel: MSQASignInButtonViewModel
+  private let action: () -> Void
+
+  /// Creates an instance of Microsoft Quick Auth sign in button.
+  /// - parameter viewModel: An instance of `MSQASignInButtonViewModel` with customization about the button's type, theme, size, etc.
+  ///     Defaults to `MSQASignInButtonViewModel` with its standard defaults.
+  /// - parameter action: A closure to be executed on button press event.
+  ///     Defaults to no-op.
+  public init(
+    viewModel: MSQASignInButtonViewModel = MSQASignInButtonViewModel(),
+    action: @escaping () -> Void = {}
```

Comment:
We want to algin with the behavior of Objective-C here, passing a `MSQASignInClient` object and call its `signInByButtonWithViewController:completionBlock` when the button clicked, like
https://github.com/microsoft/quick-authentication-mobile/blob/c3e9dd3510b7d6f6f3b157bb21a773b54e7e512c/iOS/MSQASignIn/src/MSQASignInButton.m#L422-L434

Reply:
Resolved by encapsulating related logic into the button, other changes related to this including:
* Change bundle id to be the same as the Objective-C sample app, so that redirect URI is registered.
* Add key chain access group
* Add URL schemes to info.plist

Remaining question is that:
* `signInByButtonWithViewController:completionBlock` is a private method of `MSQASignInClient`, which is not accessible by the SwiftUI button either, currently I use `signInWithViewController:completionBlock` as a workaround but I'm not quite sure if it's OK.